server/cpho/jinja2/indicators/_indicator_actions.jinja2

```diff
@@ -1,4 +1,4 @@
-<div class="dropdown">
+{# <div class="dropdown">
```

Comment:
Since this whole file is commented out, and I believe no longer used, was there a specific reason to not delete it?

Reply:
I agree, we don't need to keep this file. removing it :)